zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -108,10 +120,21 @@ public void close() {
 
   @Override
   public void export(final Record<?> record) {
+    if (writer.getBatchSize() == 0) {
+      flushLatencyMeasurement = metrics.startFlushLatencyMeasurement();
+    }
+
     writer.addRecord(record);
     lastPosition = record.getPosition();
+
     if (shouldFlush()) {
-      flush();
+      try (final var ignored = metrics.measureFlushDuration()) {
```

Comment:
👍 this looked weird at first look, until I went through the micrometer codebase to find that it returns an `AutoCloseable` and only records the duration in the `close()` method